2021-04-09 07:56:12,502 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/watchman/Gravity-Core/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
ConnectionResetError: [Errno 104] Connection reset by peer

2021-04-09 07:57:08,552 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/watchman/Gravity-Core/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-04-09 07:57:10,484 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/watchman/Gravity-Core/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

